### DISCUSSION TITLE
PHOTOPRISM_DETECT_NSFW settings

### DISCUSSION THREAD

**Discussion by tomasenskede** [photoprism/photoprism#4475] [Q&A] [answered]
I installed photoprism with this script https://tteck.github.io/Proxmox/#photoprism-lxc works great... 

but... the PHOTOPRISM_DETECT_NSFW settings... where can I set this? 

in any of these files?
hub.yml
options.yml
settings.yml

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/config-files/#feature-flags

**Reply by tomasenskede** [reply, marked as answer]
like this in options.yml

DetectNSFW: true
UploadNSFW: true

?
